### DISCUSSION TITLE
Error: Could not find App "rdns-dev" with fly.io

### DISCUSSION THREAD

**Discussion by soncms** [serverless-dns/serverless-dns#175] [Q&A]
when i run github action, it gives error like picture, how to fix it

`Run flyctl deploy --local-only --image-label f9d4d71afd099505f1bc4f56f14efe2b7c[20](https://github.com/soncms/serverless-dns/actions/runs/5420537304/jobs/9854811187#step:9:21)7cc3 --config fly.toml --strategy rolling --wait-timeout 300 --auto-confirm --no-cache --verbose 
==> Verifying app config
Validating fly.toml
WARNING: Failed to fetch platform version: Could not find App "rdns-dev"
✓ Configuration is valid
--> Verified app config
Error: Could not find App "rdns-dev"

Error: Process completed with exit code 1.`

![image](https://github.com/serverless-dns/serverless-dns/assets/87874954/4265ff02-81cd-4ae9-9c51-61ce4955f180)

**Comment by ignoramous** [comment]
Change the `FLY_APP` variable in `fly.yml`.

https://github.com/serverless-dns/serverless-dns/blob/f9d4d71afd099505f1bc4f56f14efe2b7c207cc3/.github/workflows/fly.yml#L66

You'll also have to setup `FLY_API_TOKEN` as a github-action secret.

`serverless-dns` isn't a one-click deploy to Fly. The deploy action requires changes and setup.
